ConsoleAppfForGit/Pizza/Program.cs

```diff
@@ -17,7 +17,7 @@ static void Main(string[] args)
             pizza.Ingredients.Add("ketchup");
             pizza.Ingredients.Add("sausage");
```

Comment:
Now we are adding the components right in here, but it would be cool to pass them in dynamically via args + console like ketchup salami tomato etc.